### PR TITLE
Correcting Potential Duplicate Warnings

### PR DESCRIPTION
**Pull request by rfinkley**
I corrected a duplicate warning in brands\amenity\fuel.json for "Sol". I included a brand:widkipedia tag and brand:wikidata however the wikipedia page should be updated. Sol Petroleum does not currently have it's own wikipedia page but the referenced wikipedia page links to the company's website.

### CODE REVIEW

**Comment by Identitaet** [comment]
If it doesn't have it's own Wikipedia page it's sadly not acceptable but it's got the wikidata so that's okay

Also how did you fix the duplication warning ?

**Comment by rfinkley** [comment]
> If it doesn't have it's own Wikipedia page it's sadly not acceptable but it's got the wikidata so that's okay
> 
> Also how did you fix the duplication warning ?

Thanks for letting me know about listing needing it's own page, I'll make sure to watch out for that in the future.

The potential duplicate warning went away when I added the wikidata and wikipedia tags. I ran npm run build again and did not get the warning for that item.

**Comment by rfinkley** [comment]
- Updated brand:wikidata and brand:wikipedia for Asda Café to correct duplicate warning. 

- Removed previously added brand:wikipedia reference Sol in brands/amenity/fuel.json. Sol Petroleum doesn't have a wikipedia page.

**Comment by UKChris-osm** [comment]
Hi @rfinkley 

The Wikidata entry under Sol seems to link to "_Chipotle Mexican Grill_".

**Comment by rfinkley** [comment]
> Hi @rfinkley
> 
> The Wikidata entry under Sol seems to link to "_Chipotle Mexican Grill_".

Hi @UKChris-osm, sorry, when I was reading the guide on how to update the wikidata the example showed Chipotle, I must have copied that code by mistake. I have updated the wikidata for Sol to the appropriate information. 

Sorry about that, although Chipotle is awesome.

**Review by bhousel — COMMENTED** [review]
Should this change?

File: data/brands/amenity/cafe.json [hunk 1]
```diff
@@ -69,15 +69,17 @@
...
+      "displayName": "Asda Café",
       "id": "asda-e1ef51",
       "locationSet": {"include": ["001"]},
       "tags": {
         "amenity": "cafe",
         "brand": "Asda",
         "cuisine": "coffee_shop",
-        "name": "Asda",
-        "takeaway": "yes"
+        "name": "Asda Café",
```
[the remark this answers is not in the document]
Comment (UKChris-osm) [reply data/brands/amenity/cafe.json:79]: Actually I think [Asda Café](https://www.asda.com/cafe) would be more accurate these days :)
- [Example 1](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.tripadvisor.co.uk%2FLocationPhotoDirectLink-g186430-d7624760-i122170578-Asda_Bangor_Cafe-Bangor_Gwynedd_North_Wales_Wales.html&psig=AOvVaw02iRE8nxRP3nUge0-OIGh5&ust=1602431769045000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCPCUltexquwCFQAAAAAdAAAAABAE)
- [Example 2](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F574631233682044770%2F&psig=AOvVaw02iRE8nxRP3nUge0-OIGh5&ust=1602431769045000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCPCUltexquwCFQAAAAAdAAAAABAK)

[the remark this answers is not in the document]
Comment (bhousel) [reply data/brands/amenity/cafe.json:79]: Thanks!  We are good to merge then 👍

**Review by bhousel — APPROVED** [review]

**Pull request merged**